src/components/AutocardListItem.js

```diff
@@ -25,6 +28,7 @@ const AutocardListItem = ({ card, noCardModal, children }) => {
                 <a
                   href={noCardModal ? undefined : '#'}
                   className="d-block w-100"
+                  onAuxClick={noCardModal ? undefined : e => { e.preventDefault(); if (e.button == 1) { window.open(Affiliate.getTCGLink(card)); }}}
```

Comment:
This handler is complicated enough that it should probably be broken out into a separate function.

Reply:
Thanks for the suggestion, where should I define the function?  in that same js?